android/java/org/chromium/chrome/browser/set_default_browser/BraveSetDefaultBrowserUtils.java

```diff
@@ -0,0 +1,234 @@
+/**
+ * Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.set_default_browser;
+
+import android.app.Activity;
+import android.app.role.RoleManager;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
```

Comment:
this `import android.content.SharedPreferences;` seems unused

Reply:
removed